### ISSUE TITLE
Removed traffic lights with custom ID remains in netedit

### ISSUE THREAD

**Issue by palvarezlopez** [eclipse-sumo/sumo#15251]
Their ID cannot be assigned to a new Traffic Light. Add button "remove unused traffic lights"

**Comment by namdre** [comment]
I don't think this should require an extra button. If the lights are not in the network, the ids shouldn't be blocked.
